project-codeflare/codeflare-sdk
# Cli update

# Issue link
#284 

# What changes have been made
Codeflare CLI, adding a command line interface that allows similar functionality to the Codeflare SDK. 
Changes to SDK include: 
- creating Cluster object from yaml function
- refactored some parts of the Codeflare SDK
- allow app wrapper creation to be optional

# Verification steps
-`codeflare --help` to see a list of all commands, run each one to test
- You can run unit tests with pytest

## Checks
- Testing Strategy
